src/arkode/arkode_mristep.c

```diff
@@ -1551,50 +1868,82 @@ int mriStep_TakeStep(ARKodeMem ark_mem, sunrealtype* dsmPtr, int* nflagPtr)
     case (MRISTAGE_DIRK_FAST):
       retval = mriStep_StageDIRKFast(ark_mem, step_mem, is, nflagPtr);
       break;
+    case (MRISTAGE_STIFF_ACC): retval = ARK_SUCCESS; break;
     }
     if (retval != ARK_SUCCESS) { return (retval); }
```

Comment:
I agree that I should convert an `mriStepInnerStepper_Evolve` failure to a positive [recoverable] return from `mriStep_StageERKFast`, but I don't think that we actually need a new `nflag` value.  As it currently stands, an inner stepper evolve failure sets `*nflagPtr = CONV_FAIL`.  This is because the recovery mode for a failed inner solver is essentially identical to the recovery mode for a nonlinear solver failure (cut the step and retry, up to a maximum number of failures).  Do you see any actual benefit to this, aside from allowing different `etacf` parameters?

Reply:
For understanding the behavior/performance of an implicit or ImEx method I was thinking could be helpful to differentiate between fast integration failure and an algebraic solver failure for a non-fast stage.

Reply:
Decrementing `ark_mem->ncfn` temporarily worked perfectly.  I've just pushed the new "Get" routine (and corresponding documentation), so I'll 'resolve' this comment.